app/views/genres/show.html.erb

```diff
@@ -0,0 +1,45 @@
+<% genre = params[:genre] %>
```

Comment:
The view should not need to know about the params!  The controller should grab this and pass it through.